python311Packages.opentelemetry-api: 1.22.0 -> 1.23.0; python311Packages.opentelemetry-instrumentation: 0.43b0 -> 0.44b0

natsukium opened the pull request:
## Description of changes

Diff: https://github.com/open-telemetry/opentelemetry-python/compare/refs/tags/v1.22.0...v1.23.0
Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

Diff: https://github.com/open-telemetry/opentelemetry-python-contrib/compare/refs/tags/v0.43b0...v0.44b0
Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

natsukium committed:
python311Packages.chromedb: allow local networking on darwin

de11n reviewed (approved):
Commits could be broken down a bit more, but otherwise LGTM.

natsukium committed:
python311Packages.opentelemetry-api: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-exporter-otlp: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-exporter-otlp-proto-common: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-exporter-otlp-proto-grpc: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-exporter-otlp-proto-http: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-exporter-prometheus: 1.22.0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-proto: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-sdk: 1.22.0 -> 1.23.0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-semantic-conventions: 1.22.0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-test-utils: 1.22.0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python@refs/tags/v1.22.0...v1.23.0

Changelog: https://github.com/open-telemetry/opentelemetry-python/releases/tag/refs/tags/v1.23.0

natsukium committed:
python311Packages.opentelemetry-instrumentation: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-aiohttp-client: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-asgi: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-django: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-fastapi: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-flask: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-grpc: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-instrumentation-wsgi: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0

natsukium committed:
python311Packages.opentelemetry-util-http: 0.43b0 -> 0.44b0

Diff: open-telemetry/opentelemetry-python-contrib@refs/tags/v0.43b0...v0.44b0

Changelog: https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/refs/tags/v0.44b0
